Overview.md

```diff
@@ -31,21 +31,75 @@ page_nav:
+
+#### Organizing teams
+
+With the teams feature, you can [create teams](insert link to to Create/Join a Team Guide) and organize users based on project assignments, company team structure, or any other team assignment that reflects your organizational needs. Teams are searchable by various tag criteria to help you quickly locate and assign the right people you need on your team. 
+
+![](images/overview-team-view.png)
+
+When scheduling team meetings you simply have to select the team and all team member availability is assessed to present you with the best meeting times that work for everyone.
+
+#### Merging external calendars
+
+Agenda is an API-first application that enables simplified [connections to your external scheduling applications](insert link to Connect to a Source tutorial) using the sources feature. From sources you can choose from a number of scheduling applications to [merge your meeting information](insert link to Import a Meeting tutorial) into one space. 
+
+![](images/overview-sources-view.png)
+
+Agenda unifies this data to provide you, and your team members, with a clearer view of your availability. Meetings scheduled within Agenda can also be pushed to your external scheduling applications to ensure all of your calendars are up to date.
+
+### System architecture
+
+![](images/overview-system-architecture.png)
+
+Agenda connects to several external scheduling applications and meeting platforms via APIs to ingest user meeting data. External meeting data is streamed into Agenda for processing and is used to update the meeting data set for a given user profile. Meeting information for each user is stored and used to update availability when scheduling new meetings with teams. The availability of external meeting information from connected sources allows Agenda to suggest the most appropriate meeting times based on each team member's separate schedules.
+
+For detailed information on Agenda's APIs see the [API endpoint reference guides](API.md).
+
+From a high-level Agenda is comprised of:
+
+a) Three main views that allow you to view and schedule meetings, connect to sources, and create and manage your teams.
+
+b) Data collection and business logic layer: Where all of the disparate meeting data for users is combined from external sources to provide a singular view of each users availability. This is where meeting requests are processed, updated, and made available for export back to your connected sources.
+
+c) Database: Where user's updated profile and scheduling data is stored.
+
+d) Agenda is built API-first and can be connected to a wide variety of external scheduling and meeting applications, such as: Google Calendar, Outlook, Slack, Zoom, Teams, and so on. Meeting data is then streamed from the connected source into Agenda to be processed and used to update user availability. Agenda can also be configured to send updated meeting information to selected sources, depending on the availability of their API connectors.
```

Comment:
Being built API-first is more to help developers leverage any/all of Agenda's capabilities in their own apps/programs. I think the main point to focus on here is the ability to connect with the different apps (which can be done through the API or UI).

If this were a real app, I'd imagine it would gather/store a user's authentication information for each external app, and have backend protocols for each external app it can use them to connect to. These protocols would be programmed and maintained by the Agenda development team and would basically be the core value driver of the app -- we maintain the product integrations so you don't have to.